android/build.gradle

```diff
@@ -48,5 +48,8 @@ dependencies {
         exclude group: 'org.apache.xmlgraphics', module: 'xmlgraphics-commons'
     }
     implementation 'com.google.code.gson:gson:2.8.5'
+    implementation 'androidx.localbroadcastmanager:localbroadcastmanager:1.0.0'
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    implementation "org.jetbrains.kotlinx:kotlinx-serialization-runtime:0.13.0" // JVM dependency
```

Comment:
For converting Json to a Kotlin class, will remove these in next commit. [If not needed]